pkg/pillar/cmd/zedrouter/ipaddronunet.go

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2018 Zededa, Inc.
```

Comment:
Ditto

Reply:
But more important, why move this to a different file? That makes it impossible to review your changes to the logic.